### PR TITLE
content(cms): create Resource "the-importance-of-user-centred-design-for-open-science/index"

### PR DESCRIPTION
**Pull request by LottieProvost**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by VickyGarnett** [comment]
@ttasovac @stefanprobst - This resource seems to be ready to publish, except for one issue - I can't get the video to play in the previews.  I've created a local preview as well, and I can't get it to play there either.  I've also replaced the url link with the YouTube video ID straight from YouTube to see if that was the issue, and it still won't play.  The exact error is "An error occurred. Please try again later. (Playback ID: _4qWNomXredjwmCn)".  Is this an issue on our side, or on the YouTube side?  In other words, can I go ahead and publish this.  As an FYI - the issue doesn't seem to be my machine or internet connection, as I'm able to play this video in YouTube in a separate tab without any problems.

Ta!

V

**Comment by ttasovac** [comment]
Can you paste the full youtube link which you can play in the browser please

File: content/posts/the-importance-of-user-centred-design-for-open-science/index.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+After watching this video you should:
+
+- Understand the importance of user-centred design in the Open Science perspective
+- Be able to synthesise the main phases of the iterative design process 
+- Be able to define user requirements, scenarios and personas
+- Understand the main differences and complementarities of the cognitive walkthrough method and the artefacts ecology mapping method. 
+
+
+
+<VideoCard provider="youtube" id="PpqtSWX4azI&list=PL2UW3hSUQBFOGDU6ILcVadE18wT72sMO5&index=7" title="The Importance of User-Centred Design for Open Science" subtitle="Click here to view" image="images/capture-d’écran-2022-04-05-à-16.52.02.png" />
```
Comment (stefanprobst) [inline content/posts/the-importance-of-user-centred-design-for-open-science/index.mdx]:
```suggestion
<VideoCard provider="youtube" id="PpqtSWX4azI" title="The Importance of User-Centred Design for Open Science" subtitle="Click here to view" image="images/capture-d’écran-2022-04-05-à-16.52.02.png" />
```

Comment (VickyGarnett) [reply content/posts/the-importance-of-user-centred-design-for-open-science/index.mdx]: @ttasovac https://www.youtube.com/watch?v=PpqtSWX4azI&ab_channel=TRIPLEProject, but I have the ID as "PpqtSWX4azI".

@stefanprobst You've made a suggestion that's pretty much the same as I tried in Atom, and then previewed, but I still had the same problem.  Are you able to play it in your preview?  If so, I guess I can go ahead and publish it, as it's just an issue on my end?

Comment (ttasovac) [reply content/posts/the-importance-of-user-centred-design-for-open-science/index.mdx]: you can see the preview here, but it doesn't work there either: https://dariah-campus-oy3tjvjtb-dariah.vercel.app/resource/posts/the-importance-of-user-centred-design-for-open-science 

perhaps you could tag authors to check whether they have somehow marked the video as non-embeddable or something like that on youtube?

Comment (ttasovac) [reply content/posts/the-importance-of-user-centred-design-for-open-science/index.mdx]: oh wait, the changes haven't been committed

**Comment by ttasovac** [comment]
@VickyGarnett I've committed Stefan's change. The preview is now building, once it finishes you will be able to click on the link, find the resource and make sure the video works. Let me know if it still doesn't or if you have any other problems.

**Comment by VickyGarnett** [comment]
It works!  Thanks all!  Publishing now :)

**Pull request merged**